keps/0001-kubernetes-enhancement-proposal-process.md

```diff
@@ -222,9 +222,9 @@ Metadata items:
 
 A KEP has the following states
 
-- `accepted`: The KEP has been proposed and is actively being defined.
+- `draft`: The KEP has been proposed and is actively being defined.
   This is the starting state while the KEP is being fleshed out and actively defined and discussed.
-  The owning SIG has accepted that this is work that needs to be done.
+  The owning SIG has agreed that this is work that needs to be done.
```

Comment:
even this is likely to be misread... suggest being very explicit: "The owning SIG has agreed that this is work that needs to be done, but is not ready for implementation yet."

Reply:
Sounds good. Updated.